client/src/app.ts

```diff
@@ -117,13 +117,13 @@ var obj =
     }
 );
 var json = JSON.stringify(obj);
-fetch('/updateEvent', {
-	method: 'PUT',
-	body: json, // The data
-	headers: {
-		'Content-type': 'application/json' // The type of data you're sending
-	}
-});
+// fetch('/updateEvent', {
+// 	method: 'PUT',
+// 	body: json, // The data
+// 	headers: {
+// 		'Content-type': 'application/json' // The type of data you're sending
+// 	}
+// });
```

Comment:
Removing this for now since it runs in parallel with the user vue component. Those both try to hit up Discord to check authentication, which ultimately causes discord to rate-limit the request.